### ISSUE TITLE
Missing configuration file

### ISSUE THREAD

**Issue by juliojsb** [kwilczek/collectd-cuda#1]
Hi Kamil,

Maybe I am doing something wrong, but it seems in my case collectd is unable to get the configuration file **plugins_config.sh**

I am getting the following in daemon.log (using Debian 9 OS):

`exec plugin: Unable to parse command, ignoring line: "Missing config file!`

If i just add in collectd "cd" in the script **collectd_cuda.sh** to the path where the config file is located everything works:

`cd /home/jota/my-scripts/collectd/`

Regards,

**Comment by kwilczek** [comment]
Hi @juliojsb,

you are right, there was a relative path bug in my script which prevented the configuration
file from being sourced. I already pushed the updated version which will also check for the
presence of `nvidia-smi` tool (I'm currently on vacation and had no access to a machine with
a nVidia card -- I tested this on a Vagrant VM, but it was enough to catch some more small
issues :))

PS. You might consider enabling `logfile` plugin which redirects `collectd` logs to a separate
file (for example `/var/log/collectd.log`. This keeps logs cleaner and allows for much easier
debugging.

Best Regards,
Kamil
